esrally/exceptions.py

```diff
@@ -25,16 +25,10 @@ class RallyError(Exception):
     """
 
     def __init__(self, message, cause=None):
-        super().__init__(message, cause)
+        super().__init__(message)
         self.message = message
         self.cause = cause
 
-    def __repr__(self):
-        return self.message
-
-    def __str__(self):
-        return self.message
-
```

Comment:
Oh my god, this is really cool. This will solve a LOT of issues we had with unprintable exceptions that I never figured out. (I wonder why we're not using __cause__ by the way)